README.md

```diff
@@ -7,50 +10,72 @@ To add this library to your model, add the following lines to the top of your ag
+--------------------------------------------------------------------------------------- | --------------------------------------
+[AWSSNS_ACTION_CONFIRM_SUBSCRIPTION](#awssns_action_confirm_subscription)               | Verifies an endpoint owner's intent to receive messages
+[AWSSNS_ACTION_LIST_SUBSCRIPTIONS](#awssns_action_list_subscriptions)                   | Returns a xml list of the requester's subscriptions
+[AWSSNS_ACTION_LIST_SUBSCRIPTIONS_BY_TOPIC](#awssns_action_list_subscriptions_by_topic) | Returns a xml list of the subscriptions to a specific topic
+[AWSSNS_ACTION_LIST_TOPICS](#awssns_action_list_topics)                                 | Returns a xml list of the requester's topics
+[AWSSNS_ACTION_PUBLISH](#awssns_action_publish)                                         | Sends a message to an Amazon SNS topic
+[AWSSNS_ACTION_SUBSCRIBE](#awssns_action_subscribe)                                     | Prepares to subscribe to an endpoint
+[AWSSNS_ACTION_UNSUBSCRIBE](#awssns_action_unsubscribe)                                 | Deletes a subscription
 
-where `params` includes
+#### Action parameters
+
+##### AWSSNS_ACTION_CONFIRM_SUBSCRIPTION
+Verifies an endpoint owner's intent to receive messages by validating the token sent to the endpoint by an earlier Subscribe action.
+Please view the [AWS SNS documentation](http://docs.aws.amazon.com/sns/latest/api/API_ConfirmSubscription.html) for more information.
 
- Parameter 	              | Type    | Required | Default | Description             
+##### Action parameters ([`params`](#actionactiontype-params-cb) argument)
+Parameter 	              | Type    | Required | Default | Description             
 ------------------------- | ------- | -------- | ------- | -------------------
-AuthenticateOnUnsubscribe | String  | No       | null    | Disallows unauthenticated unsubscribes of the subscription. If the value of this parameter is true and the request has an AWS signature, then only the topic owner and the subscription owner can unsubscribe the endpoint. The unsubscribe action requires AWS authentication
-Token					  | String  | Yes      | N/A     | Short-lived token sent to an endpoint during the Subscribe action
-TopicArn                  | String  | Yes      | N/A     | The ARN of the topic for which you wish to confirm a subscription
+AuthenticateOnUnsubscribe | string  | No       | null    | Disallows unauthenticated unsubscribes of the subscription. If the value of this parameter is true and the request has an AWS signature, then only the topic owner and the subscription owner can unsubscribe the endpoint. The unsubscribe action requires AWS authentication
+Token					  | string  | Yes      | N/A     | Short-lived token sent to an endpoint during the Subscribe action
+TopicArn                  | string  | Yes      | N/A     | The ARN of the topic for which you wish to confirm a subscription
 
 #### Example
```

Comment:
Example's should be renamed to reflect their action.
in this case ##### Confirm Subscription Example
needs to be done for all examples